src/assets/stylesheets/components/_alert.scss

```
@@ -0,0 +1,56 @@
+.alert {
+  display: flex;
+  width: 100%;
+  margin-bottom: 2rem;
+
+  animation: slide-in 0.2s ease-in;
+
+  border-radius: 12px;
+
+  background: rgba(37, 37, 37, 0.6);
+
+  color: #fff;
+  backdrop-filter: blur(27.1828px);
+  @keyframes slide-in {
+    0% {
+      transform: skewX(10deg) translateX(-25px);
+    }
+
+    100% {
+      transform: skewX(0deg);
+    }
+  }
+
+  .alert-icon {
```

It's part of our convention to [follow BEM](https://nimblehq.co/compass/development/code-conventions/css/#convention-75) so we can keep a clear separation between the HTML and the CSS

We can use `@extend` if we want to retain the original styling from bootstrap 😄 

```suggestion
  &__icon {
    @extend .alert-icon

```

---

The class name is changed on https://github.com/suphanatjarukulgowit/react-nimble-survey/commit/f5c1469821f5cd51510a802528ec903b4dd6df28 👍